dluhc-component-library/src/components/timetableForm/components/LandingPage.tsx

```diff
@@ -24,9 +31,27 @@ const LandingPage = ({ onContinueClick }: LandingPageProps) => {
         </button>
       </div>
       <div>
-        <button className="bg-green-700 hover:bg-green-800 text-white py-1 px-2">
+        <label
+          type="button"
+          role="button"
+          className="bg-green-700 hover:bg-green-800 text-white py-1 px-2"
+        >
           Upload and edit an exisiting timetable
-        </button>
+          <input
+            hidden
+            type="file"
+            accept="text/json"
+            onChange={async (event) => {
```

Comment:
Can we pull this out into a function please. There is too much here to be in your html in my opinion